_stbt/match.py

```diff
@@ -751,6 +753,12 @@ def _log_match_image_debug(imglog):
             result.region, _Annotation.MATCHED if result._first_pass_matched  # pylint:disable=protected-access
             else _Annotation.NO_MATCH)
 
+    imglog.imwrite(
+        "source_with_matches", imglog.images["source"],
+        [x.region for x in imglog.data["matches"]],
+        [_Annotation.MATCHED if x.match else _Annotation.NO_MATCH
+         for x in imglog.data["matches"]])
```

Comment:
We could use `div`s here to draw the regions so we don't need to write `source_with_matches.png`